easybuild/scripts/rpath_args.py

```diff
@@ -93,6 +93,8 @@
 
     # filter out --enable-new-dtags if it's used;
```

Comment:
it fixes the cases where the command-line consists of '... -Wl, --enable-new-dtags -fomit-frame-pointer'. In the current version, rpath_args.py would modify this into '-Wl, -fomit-frame-pointer' and the option -fomit-frame-pointer is passed to the linker, which fails because it is an unknown option. The fix just passes the same option --disable-new-dtags multiple times to the linker, which shouldn't be a problem.